types/probe-image-size/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for probe-image-size 5.0
+// Project: https://github.com/nodeca/probe-image-size#readme
+// Definitions by: Jinesh Shah <https://github.com/jineshshah36>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function probe(source: string, opts?: probe.ProbeOptions): Promise<probe.ProbeResult>;
```

Comment:
what about legacy options documented in the readme and in the source code:
https://github.com/nodeca/probe-image-size/blob/master/index.js#L35

> Note 2. We still support legacy v2.x signature for http probe (src is Object as described in request). But it will be deprecated in next versions.

Reply:
Apologies, I didn't realize we were required to provide full type defs for the module. I can make changes and update

Reply:
> I didn't realize we were required to provide full type defs for the module. I can make changes and update

No, we are not. Would be nice to have them in first place (that is the exported function that people can start using strigh away after installing package and playing around). But that's OK to provide that as an update (anyone can do this).

Reply:
Ok thanks!